# Exceptions.py

class PersonajeNoEncontrado(Exception):
    pass

class MisionNoDisponible(Exception):
    pass
